### ISSUE TITLE
[BUG/ISSUE] (minor issue) Initialize CO2 to a relevant value in fullchem_mod.F90 

### ISSUE THREAD

**Issue by msl3v** [geoschem/geos-chem#1529]
Hello all.

We've been looking at the process of error estimation in KPP's Rosenbrock solver, and noticed that in calculating the error norm of a FullChem run, the relative error of CO2 is large. There are several points this raises:
1) The error norm determines the internal substep size. 
2) CO2 is not a stiff or active species. It is essentially for mass balance of carbon oxidation. It is nowhere a reactant.
3) The reason the relative error for CO2 is high is because it is initialized to zero for every timestep. 
4) the error norm should be minimized: non-relevant terms should be excluded or their impacts kept small.

This issue can be resolved by simply setting CO2 to a relevant (or even arbitrarily) large value. Pardon me if I don't propose a number here, because C(ind_CO2) is in molec/cm3, and would be pressure dependent if it were calculated appropriately.

In fullchem_mod.F90, where CO2 is set to zero, there is a 2016 comment from @msulprizio 
```
       ! Temporary fix for CO2                                                                                                                                                                                                                
       ! CO2 is a dead species and needs to be set to zero to                                                                                                                                                                                 
       ! match the old SMVGEAR code (mps, 6/14/16)                                                                                                                                                                                            
       IF ( TRIM( SpcInfo%Name ) == 'CO2' ) THEN
          State_Chm%Species(N)%Conc(:,:,:) = 0.0_fp
       ENDIF
```
... this was right after completing the 'flexchem' upgrade. I suspect we no longer need to worry about SMVGEAR behavior.

tagging @christophkeller 

This is also related to ongoing [KPP work:](https://github.com/KineticPreProcessor/KPP/issues/66)

**Comment by yantosca** [comment]
Thanks @msl3v.  This looks like some leftover baggage.

**Comment by yantosca** [comment]
@msl3v, should we use the initial value from the restart file?

**Comment by msl3v** [comment]
@yantosca, It looks like CO2 is reset to zero every timestep;  and we want a big number. The bigger the initial value of CO2 at the beginning of the integration, the smaller CO2's contribution to relative error. I think there's two options:
1) Run a benchmark where C(ind_CO2) isn't set to zero, and use the restart CO2 field from that. This would accumulate CO2.
2) Simply set CO2 to an appropriate ambient value, e.g. 400 ppm (converted to mcl/cm3)

(2) would do more to minimize the CO2 term in the error norm since it is a much bigger number than that from VOC oxidation.

**Comment by msulprizio** [comment]
@msl3v Do you have a fix for this? If so, could you submit a PR?

I tried implementing the suggested fix 2 above, but it resulted in KPP errors (even after fixing the units).

**Comment by yantosca** [comment]
Closed by #1731

**Comment by msulprizio** [comment]
The fix for this (PR #1731) has now been merged into the 14.2.0 development branch.
